### ISSUE TITLE
do you consider to use a more powerful model, CogVLM as the backend?

### ISSUE THREAD

**Issue by Sleepychord** [lxe/llavavision#1]
https://github.com/THUDM/CogVLM is an open-source model we released, with demo. It performs better than LLaVA in most cases.

**Comment by lxe** [comment]
Yeah I'm playing with CogVLM now. It's definitely more hefty and slower.
